swift/ql/src/queries/Security/CWE-328/WeakPasswordHashing.qhelp

```diff
@@ -80,20 +82,11 @@
                 Cheat Sheet
             </a>
         </li>
-        <li>
-            nuget: <a href="https://www.nuget.org/packages/Konscious.Security.Cryptography.Argon2/">Konscious.Security.Cryptography.Argon2</a>
-        </li>
-        <li>
-            nuget: <a href="https://www.nuget.org/packages/Isopoh.Cryptography.Argon2">Isopoh.Cryptography.Argon</a>
-        </li>
         <li>
             libsodium: <a href="https://doc.libsodium.org/bindings_for_other_languages#bindings-programming-languages">libsodium bindings for other languages</a>
         </li>
         <li>
-            nuget: <a href="https://www.nuget.org/packages/BCrypt.Net-Core/">BCrypt.Net-Core</a>
-        </li>
-        <li>
-            nuget: <a href="https://www.nuget.org/packages/CryptSharpOfficial/">Scrypt and PBKDF2</a>
+            GitHub: <a href="https://github.com/tmthecoder/Argon2Swift">Argon2Swift</a>
```

Comment:
Are there additional libraries we should be recommending here? In particular bcrypt and/or scrypt libraries.